reducers/views/search.ts

```diff
@@ -15,7 +17,9 @@ function modalSearch(state = '', action) {
     }
 }
 
-function modalFilters(state = {}, action) {
+type ModalFiltersState = {roles?: string[]; channel_roles?: string[]; team_roles?: string[]};
+
+function modalFilters(state: ModalFiltersState = {}, action: GenericAction) {
```

Comment:
Getting the type from the store type would definitely help here

Reply:
Done